simple-page-ordering.php

```diff
@@ -168,10 +168,26 @@ public static function load_edit_screen() {
 					'sort_by_order_link',
 				)
 			);
+			add_action( 'pre_get_posts', array( __CLASS__, 'filter_query' ) );
 			add_action( 'wp', array( __CLASS__, 'wp' ) );
 			add_action( 'admin_head', array( __CLASS__, 'admin_head' ) );
 		}
 
+		/**
+		 * This is to enable pagination.
+		 *
+		 * @param WP_Query $query The WP_Query instance (passed by reference).
+		 */
+		public static function filter_query( $query ) {
+			$is_simple_page_ordering = isset( $_GET['id'] ) ? 'simple-page-ordering' === sanitize_text_field( wp_unslash( $_GET['id'] ) ) : false;
```

Comment:
No need to sanitize or unslash when comparing to a string that requires neither.

```suggestion
			// phpcs:ignore WordPress.Security.NonceVerification.Recommended
			$is_simple_page_ordering = isset( $_GET['id'] ) ? 'simple-page-ordering' === $_GET['id'] : false;
```